### PR TITLE
Hackathon Fixes

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@lamagnifica  some cleanup in LTI file; added note on line 166

**Comment by mhoeber** [comment]
@lamagnifica  html file line 413 is new paragraph for doc-305

**Comment by mhoeber** [comment]
@lamagnifica   made edits and added several other files.

File: en_us/shared/building_and_running_chapters/building_course/course_rerun.rst [hunk 1]
```diff
@@ -50,6 +50,12 @@ any way.
    * - Wiki contributions
      - No.
 
+.. caution::
+ After you re-run a course, modifications to the original course and the re-run
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/course_rerun.rst]: does this need to be a caution? just curious

File: en_us/shared/building_and_running_chapters/building_course/course_rerun.rst [hunk 2]
```diff
@@ -50,6 +50,12 @@ any way.
...
+ course are independent. Changes to either course are not propagated to the
+ other course. Therefore you should ensure that the original course content is
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/course_rerun.rst]: Therefore, (insert comma)

File: en_us/shared/building_and_running_chapters/building_course/course_rerun.rst [hunk 3]
```diff
@@ -165,6 +175,8 @@ through the course and verify that it is ready for release. You can also refer
 to the :ref:`Launch` chapter for tools and ideas that help you prepare the
 course for launch.
 
+.. note:: 
+  Changes you make in the new course are not propagated to the original course.
 
 .. _edge.edx.org: http://edge.edx.org
-.. _edx.org: http://edx.org
\ No newline at end of file
+.. _edx.org: http://edx.o
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/course_rerun.rst]: ??

File: en_us/shared/building_and_running_chapters/building_course/export_import_course.rst [hunk 4]
```diff
@@ -92,3 +98,9 @@ To import a course:
 #. Locate the file that you want, and then click **Open**.
 #. Click **Replace my course with the one above**.
 
+.. note:: 
+ When you import a course, important dates such as the course start date and
+ time, are overwritten. After the import is complete, you should check
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/export_import_course.rst]: add another comma after "important dates"

File: en_us/shared/building_and_running_chapters/building_course/export_import_course.rst [hunk 5]
```diff
@@ -92,3 +98,9 @@ To import a course:
...
+ important dates to ensure they are set as intended. For more information, see
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/export_import_course.rst]: "important" seems a little overused, though maybe it's the proximity with multiple uses of "import". Strike this second "important"?

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 6]
```diff
@@ -81,11 +75,13 @@ In Studio, from the **Settings** menu, select **Schedule and Details**.
 
 Follow the on-screen text to enter the course and enrollment schedule.
 
+We recommend that you check that all important dates are correct one week
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: style guide question: should we all use the editorial "we" like this to refer to the edX staff, or say that "edX recommends"?

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 7]
```diff
@@ -99,11 +95,15 @@ The Course Start Date
...
-Topics`.
+summary page). Students can see some parts of the course before the course
+start date. For example, students can see your **Course Info** page and course-
+wide discussion topics as soon as they enroll in your course. For more
+information about course-wide discussion topics, see :ref:`Create CourseWide
+Discussion Topics`.
+
+We recommend that you set the start time of your course to early in the
+day, generally 12:00 UTC or earlier. Learners often expect the course to be
+available on the start date and try to access course content during the day.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: ...on the start date ^in their own time zones^ and try... (?)

File: en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst [hunk 8]
```diff
@@ -108,15 +108,24 @@ descriptions.
...
 #. Choose a font family for selected text, such as Arial, Courier New, or Times
    New Roman.
-#. Format selected text in bold.
-#. Format selected text in italics.
-#. Underline selected text.
-#. Apply a color to the selected text.
-#. Format selected text as code.
-#. Create a bulleted list.
-#. Create a numbered list.
+#. Format selected text in bold. The editor inserts ``<strong>`` tags around
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst:111]: yay!

File: en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst [hunk 9]
```diff
@@ -108,15 +108,24 @@ descriptions.
...
+   the selected text.
+#. Format selected text in italics. The editor inserts ``<em>`` tags around the
+   selected text.
+#. Underline the selected text. The editor encloses the selected text in
+   the tag ``<span style="text-decoration: underline;">``.
+#. Apply a color to the selected text. The editor encloses the selected text in
+   the tag ``<span style="color: color-hex-code;">``.
+#. Format selected text as a code block. The editor inserts ``<code>`` tags
+   around the selected text, which is then displayed in a monospace font.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst:120]: which is then displayed --> which then displays (?)

File: en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst [hunk 10]
```diff
@@ -108,15 +108,24 @@ descriptions.
...
+#. Create a bulleted list. The editor inserts ``<ul>`` tags
+   around the selected text, and encloses each paragraph in ``<li>`` tags.
+#. Create a numbered list. The editor inserts ``<ol>`` tags
+   around the selected text, and encloses each paragraph in ``<li>`` tags.
 #. Decrease and increased the indentation of the selected paragraph.
-#. Format the selected paragraph as a blockquote.
+#. Format the selected paragraph as a blockquote. The editor inserts
+   ``<blockquote>`` tags around the selected text, which is then displayed as a
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst:127]: ditto

File: en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst [hunk 11]
```diff
@@ -334,8 +349,12 @@ You can add a link in an HTML component to any file that is uploaded for the
 course. For more information about uploading files, see :ref:`Add Files to a
 Course`.
 
-#. On the **Files & Uploads** page, copy the **Studio** URL of the file.
+.. tip::
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/creating_content/create_html_component.rst:352]: tip! I love tips! I didn't know we had this markup!

**Comment by lamagnifica** [comment]
some more nits. this is so great. :+1:

**Pull request merged**